src/asserts/string-of-length-assert.js

```diff
@@ -0,0 +1,28 @@
+
+/**
+ * Module dependencies.
+ */
+
+import { Assert as is } from 'validator.js';
+
+/**
+ * Exports `StringOfLengthAssert`.
+ */
+
+export default function stringOfLengthAssert({ max, min = 1 } = {}) {
+  this.max = max;
```

Comment:
Shouldn't be needed, AFAIK, because all violations this assert throws are violations from other asserts, therefore, it would never expose the value of its own `__class__`.